Makefile

```diff
@@ -3,13 +3,15 @@ TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
 include ./Makefile.os
 
 GITHUB_VERSION ?= main
+OAUTH_TAG = main
```

Comment:
We have `GITHUB_VERSION`, `RELEASE_VERSION` ... I think this should be named `OAUTH_VERSION`.